src/wechaty/user/contact.py

```diff
@@ -58,7 +60,7 @@
 
 
 # pylint:disable=R0904
-class Contact(Accessory, AsyncIOEventEmitter):
+class Contact(Accessory[ContactPayload], AsyncIOEventEmitter):
```

Comment:
This generic type info (`ContactPayload`) should not be here.  There is a little excessive design in my opinion.